### DISCUSSION TITLE
Magic Eden wallet request/question

### DISCUSSION THREAD

**Discussion by amarcu** [rainbow-me/rainbowkit#1887] [Q&A]
Would be great if magic eden wallet would be natively supported, I could not find any PRs / branches with an integration.
Or if I can get a similar wallet integration I can work my way into integrating it.
Wagmi wallet inject logic: https://docs.magiceden.io/docs/me-wallet-on-evm-with-wagmi
https://wallet.magiceden.io/

**Comment by 0xmkh** [comment]
@amarcu Does Magic Eden wallet support EVM chains only ?

**Reply by amarcu** [reply]
Also supports Solana(SVM) and Bitcoin.

From the docs:
`The Magic Eden wallet can be integrated into dApps by using trusted libraries, such as the Solana Wallet Adapter on the SVM, or WAGMI on the EVM. Alternatively, web apps can interact individually with the Magic Eden wallet by accessing the magicEden provider that is injected by the ME wallet extension`

**Reply by 0xmkh** [reply]
Nice! Btw is there a way to disable Solana and Bitcoin on their wallet during connection ? That way we don't get unexpected behaviours from viem and wagmi.

**Reply by amarcu** [reply]
From what i see, the solana wallet uses the [solana wallet adapter](https://github.com/anza-xyz/wallet-adapter) and for bitcoin it's based on [sats-connect](https://docs.xverse.app/sats-connect)
Looks like each integration is different so they are essentially different wallets but I might not understand everything properly.

**Reply by 0xmkh** [reply]
Sounds good. Will take a look shortly 🙏

**Reply by 0xmkh** [reply]
Seems like their wallet crashes with the latest wagmi version:

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/b6b75102-7c47-4493-81f8-e3e65277694f)

**Reply by amarcu** [reply]
Thanks for the update, have contacted the magic eden team and they are looking into the issue.
